bucket/coppeliasim.json

```diff
@@ -23,13 +23,13 @@
         ]
     ],
     "checkver": {
-        "url": "https://www.coppeliarobotics.com/helpFiles/en/versionInfo.htm",
-        "regex": "CoppeliaSim\\s+V([\\d.]+)\\s+\\((?!not yet)"
+        "url": "https://www.coppeliarobotics.com/downloads",
+        "regex": "CoppeliaSim_Edu_V([\\d_]+)_rev(?<rev>\\d+)_Win.zip"
```

Comment:
We should not hardcode `revX`.

Reply:
https://github.com/StarsbySea/Extras/blob/02454e4fc89dd864bf54d4242b1c5c241a49d47d/bucket/coppeliasim.json#L27

I have made some changes as per your request, please see.

But from this page https://www.coppeliarobotics.com/previousVersions, recent versions contain the rev field that's all.

Reply:
`scoop install/download coppeliasim@4.3.0` broken when hardcode it.